test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -942,6 +942,32 @@ def forward(self, x):
             loaded_exported_program, (x,), skip_dynamic_shapes_check=True
         )
 
+    @pytorch_test_common.xfail_if_model_type_is_not_exportedprogram(
+        "Unsupported FX nodes: {'call_function': ['aten.add_.Tensor']}. "
```

Comment:
@BowenBao should Functionalization take care of this?

Reply:
Looks like discrepency between our Functionalization and the one inside torch.export.export?

Reply:
Make sure we have an issue tracking this.

Reply:
https://github.com/pytorch/pytorch/issues/114406

Reply:
yea because dynamo_export was not lifting buffers, there was another issue you can link it there too.